test/controllers/slides_controller_test.rb

```diff
@@ -143,4 +143,12 @@ class SlidesControllerTest < ActionController::TestCase
 
     assert_equal "horizontal", slide.reload.orientation
   end
+
+  test "assert that `controls muted` attribute is present" do
+    video_foreground = slides.foreground_type
+    video_background = slides.background_type
+
+    assert_equal("video controls muted", slide.foreground_type)
+    assert_equal("video controls muted", slide.background_type)
```

Comment:
I noticed I made a mistake here and used `slide.foreground_type` instead of `video_foreground` and the same for the background, but I still am not sure how best to ID them on lines 148 and 149.